### PR TITLE
Cbrelease 4.8.11

### PR DESCRIPTION
**Pull request by venkykandagaddala**

### CODE REVIEW

File: src/app/component/in-sight-side-bar/in-sight-side-bar.component.html [hunk 1]
```diff
@@ -4,10 +4,87 @@
...
+                        <div class="flex flex-middle copy-items">
+                            <span class="pr-2 custom-width">{{assessmentsData.userName}}</span>
+                            <span class=""><mat-icon class="cursor-pointer copy-icon" (click)="copyToClipboard('Venkat')">content_copy</mat-icon></span>
+                        </div>
+                    </div>
+                    <div class="flex flex-middle" style="padding: 5px 0px;">
+                        <span class="pr-6 font-semibold">Password</span>
+                        <div class="flex flex-middle copy-items">
+                            <span class="pr-2  custom-width">{{assessmentsData.password}}</span>
+                            <span class=""><mat-icon class="cursor-pointer copy-icon" (click)="copyToClipboard('Test@1234')">content_copy</mat-icon></span>
```
Comment (vishnubansaltarento) [inline src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:53]: are we showing user name and password here?

File: library/ws-widget/collection/src/lib/profile-v2/profile-certificate-dialog/profile-certificate-dialog.component.ts [hunk 2]
```diff
@@ -22,6 +23,7 @@ export class ProfileCertificateDialogComponent implements OnInit {
   author!: string
   userID: any
   courseData: any
+  environment!: any
 
   navUrl: any = ''
   shareUrl = 'https://medium.com/@garfunkel61/angular-simplest-solution-for-social-sharing-feature-6f00d5d99c5e'
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/profile-v2/profile-certificate-dialog/profile-certificate-dialog.component.ts:29]: are we using this shareurl?

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html [hunk 3]
```diff
@@ -239,6 +239,53 @@
                 [isLoading]="insightsDataLoading"></ws-widget-weekly-claps>
             </div>
 
+            <div class="assessments-wrapper" *ngIf="assessmentsData">
+              <div class="assessments-info-wrapper">
+                  <div class="flex justify-between">
+                      <div class="flex flex-col">
+                          <div class="flex flex-between flex-middle">
+                              <p class="flex-align-start upcoming-assessment-title flex flex-middle margin-remove">Upcoming Assessment
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html:247]: please apply transalation for the same

File: src/app/component/in-sight-side-bar/in-sight-side-bar.component.html [hunk 4]
```diff
@@ -4,10 +4,87 @@
...
+                    <div class="flex flex-col">
+                        <div class="flex flex-between flex-middle">
+                            <p class="flex-align-start upcoming-assessment-title flex flex-middle margin-remove">Upcoming Assessment
+                                <img class="margin-left-s cursor-pointer" width="20px" height="20px"
+                                    src="assets/icons/home/help_icon.svg" #tooltip="matTooltip" [matTooltipClass]="'below'"
+                                    matTooltip="View your Credentials and take your Assessment.">
+                            </p>
+                        </div>
+                        <button
+                            class="bg-blue-700 text-sm font-semibold text-white py-3 px-5 rounded-full border-none cursor-pointer mt-4 mb-2 tracking-wide"
```
Comment (vishnubansaltarento) [inline src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:24]: please do transaltion fo hardcoded text

File: src/app/component/in-sight-side-bar/in-sight-side-bar.component.html [hunk 5]
```diff
@@ -4,10 +4,87 @@
...
+    <div *ngIf="assessmentsData" class="margin-bottom-m" class="collapsible" [@collapsDiscuss]="collapsed" [ngClass]="{'padding-bottom-m': !collapsed}">
+        <div class="assessments-wrapper">
+            <div class="assessments-info-wrapper">
+                <div class="flex justify-between">
+                    <div class="flex flex-col">
+                        <div class="flex flex-between flex-middle">
+                            <p class="flex-align-start upcoming-assessment-title flex flex-middle margin-remove">Upcoming Assessment
+                                <img class="margin-left-s cursor-pointer" width="20px" height="20px"
+                                    src="assets/icons/home/help_icon.svg" #tooltip="matTooltip" [matTooltipClass]="'below'"
+                                    matTooltip="View your Credentials and take your Assessment.">
```
Comment (vishnubansaltarento) [inline src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:20]: please do transaltion for hard coded text

Comment (venkykandagaddala) [reply src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:20]: We commented on this while enabling I can do the transition.

File: src/app/component/in-sight-side-bar/in-sight-side-bar.component.html (hunk 4 above)
Comment (venkykandagaddala) [reply src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:24]: We commented on this while enabling I can do the transition.

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html (hunk 3 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html:247]: We commented on this while enabling I can do the transition.

File: library/ws-widget/collection/src/lib/profile-v2/profile-certificate-dialog/profile-certificate-dialog.component.ts (hunk 2 above)
Comment (venkykandagaddala) [reply library/ws-widget/collection/src/lib/profile-v2/profile-certificate-dialog/profile-certificate-dialog.component.ts:29]: that is existing code, I am not sure we are using that.

File: src/app/component/in-sight-side-bar/in-sight-side-bar.component.html (hunk 1 above)
Comment (venkykandagaddala) [reply src/app/component/in-sight-side-bar/in-sight-side-bar.component.html:53]: We commented on this while enabling I can do the transition.

**Pull request merged**